x-pack/solutions/observability/plugins/observability/public/pages/alert_details/alert_details.tsx

```diff
@@ -205,6 +205,10 @@ export function AlertDetails() {
   usePageReady({
     isRefreshing: isLoading,
     isReady: !isLoading && !!alertDetail && activeTabId === 'overview',
+    meta: {
+      description:
+        '[ttfmp_alerts] The Observability Alert Details overview page has loaded successfully',
```

Comment:
Can we please also have a separate key for this page: `ttfmp_alert_details`? Since this is a pageId, it makes sense to have separate pageIds for alert details and the alerts pages.

Reply:
See [2291054](https://github.com/elastic/kibana/pull/225095/commits/2291054fcc11c442282ee59250518db81536b297).